ftp-ballerina/Module.md

```diff
@@ -0,0 +1,214 @@
+## Overview
+
+This module provides an FTP client and an FTP server listener implementation to facilitate an FTP connection to a remote location.
+
+### FTP Client
+
+The `ftp:Client` connects to an FTP server and performs various operations on the files. Currently, it supports the
+generic FTP operations; `get`, `delete`, `put`, `append`, `mkdir`, `rmdir`, `isDirectory`, `rename`, `size`, and
+ `list`.
+
+An FTP client endpoint is defined using the parameters `protocol` and `host`, and optionally the `port` and
+`secureSocket`. Authentication configuration can be configured using the `secureSocket` parameter for basicAuth,
+private key, or TrustStore/Keystore.
+
+##### Creating a client
+
+The following code creates an FTP client and perform I/O operations, which connects to the FTP server with Basic Auth.
+```ballerina
+import ballerina/ftp;
+
```

Comment:
we don't need an import statement. wdyt?

Reply:
Yeah. Removed in all the import occurrences.